TUMFARSynchrony/experimental-hub
# Development without dlib for merge

This is a draft Pull Request Template. Please fill out the neccessary details below

# Overview
This PR contains changes related to dockerized OpenFace. 

## Changes
OpenFace AU Filter has removed and rather than that now we have an interface for OpenFace which inherited by 2 different classes which are OpenFaceWithDocker and OpenFaceWithSubprocess. It can be selected by the developer from backend/config.json (**open_face_type** variable with **subprocess** or **dockerized** values).

## How to test?
Launch the application and see that with subprocess in open_face_type variable works smoothly.

# Checklist
